spynnaker/pyNN/models/neuron/abstract_population_vertex.py

```diff
@@ -317,7 +315,7 @@ def get_sdram_usage_for_neuron_params(self, vertex_slice):
         :return: The SDRAM required for the neuron region
         """
         return (
-            self._BYTES_TILL_START_OF_GLOBAL_PARAMETERS +
+            self.BYTES_TILL_START_OF_GLOBAL_PARAMETERS +
```

Comment:
I could see that this whole method would make a little bit more sense as part of the machine vertex.  That said, I can see the issue with having to ask the neuron impl for its sdram usage, making it harder.  The property could still come from the machine vertex though I think...

Reply:
goes against previous logic

Reply:
OK, fair enough